common/const.go

```diff
@@ -60,5 +60,7 @@ const (
 	CiliumCHeaderPrefix = "CILIUM_BASE64_"
 
 	// CiliumIdentityAnnotation is the annotation key used to map to an endpoint's security identity.
-	CiliumIdentityAnnotation = "cilium-identity"
+	CiliumIdentityAnnotation = "cilium.io.identity"
```

Comment:
Shouldn't this be `cilium.io/identity`?

Looking at other annotation examples:
`annotation.kubernetes.io/config.seen`